apps/rush-lib/src/start.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { EOL } from 'os';
+import * as colors from 'colors';
+
+import rushVersion from './rushVersion';
+import RushCommandLineParser from './cli/actions/RushCommandLineParser';
+
+// tslint:disable-next-line:export-name
+export function executeCli(wrapperVersion: string, isManaged: boolean = false): void {
```

Comment:
> isManaged: boolean = false [](start = 50, length = 27)

This is a formal plug-in contract; nothing should be optional

Reply:
I'd like to limit the number of arguments, but I'll make the second one non-optional.

---
In reply to: [145844452](https://github.com/Microsoft/web-build-tools/pull/397#discussion_r145844452) [](ancestors = 145844452)